src/js/components/Button/__tests__/Button-kind-test.js

```diff
@@ -37,6 +39,46 @@ describe('Button kind', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('button with label and icon', () => {
+    const { container } = render(
+      <Grommet
+        theme={{
+          button: {
+            default: {
+              color: '#FFF',
+              background: '#000',
+            },
+          },
+        }}
+      >
+        <Button label="label" icon={<Add />} hoverIndicator />
```

Comment:
The docs note that `hoverIndicator` should only be used with `plain` buttons, but this isn't a plain button. Do we still want to have a test case?

Reply:
also if this test just has a single expect, i'm not sure we are capturing the `hoverIndicator` in the snapshot

Reply:
Good catch, the `hoverIndicator` slipped in and is not needed for this test, let me clean it up.